adaptive-learning/README.md

```diff
@@ -0,0 +1,64 @@
+# Adaptive Learning  of Object Detection Models
+
+
+The goal of adaptive learning is to build robust systems that can work efficiently and adapt to new environments (novel data distributions) without labeled data or manual labeling or storing or transferring data from the edge devices to other systems. This framework will dramatically simplify machine learning systems’ architecture and lower their deployment and maintenance costs when applied.
+We published a comprehensive blog about the concept and features of Adaptive Learning. Please refer to the [blog post](https://neuralet.com/article/adaptive-learning-part-1/) for more details.
+
+## prerequisites
+1. At least one Nvidia GPU
+2. At least 6GB of RAM
+3. [Docker](https://docs.docker.com/get-docker/)
+4. [Docker Compose](https://github.com/docker/compose)
+5. [Nvidia Docker Toolkit](https://github.com/NVIDIA/nvidia-docker)
+
+## Getting Started
+### Clone Neuralet Repository
+Make sure you have the prerequisites and then clone this repository to your local system by running this command:
+```
+git clone https://github.com/neuralet/neuralet.git
```

Comment:
I saw you added several files for adaptive learning. Is it still necessary to clone from neuralet/neuralet?

Reply:
No, it was just the copy of the original README, I changed it and referred to the original one for more details.